app/views/events/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if flash[alert] %>
```

Comment:
- [ ] Make sure that any view logic, loops and conditionals, is kept inside helper methods.